DEFAULT_CONFIG.json5

```diff
@@ -440,14 +440,14 @@
           /// then amount of memory being allocated for each queue is SIZE_XXX * LINK_MTU.
           /// If qos is false, then only the DATA priority will be allocated.
           size: {
-            control: 1,
-            real_time: 1,
-            interactive_high: 1,
-            interactive_low: 1,
+            control: 2,
+            real_time: 2,
+            interactive_high: 2,
+            interactive_low: 2,
             data_high: 2,
-            data: 4,
-            data_low: 4,
-            background: 4,
+            data: 2,
```

Comment:
I think this would impact maximum throughput by a lot.

Reply:
In my tests I have observed no difference whatsoever... do you observe the same?